samples/ClassroomLabs/Scripts/CustomContribRole/ImportCustomContribRole.ps1

```diff
@@ -0,0 +1,8 @@
+$index = (Get-Content -path ".\AzLabsCustomContribRole.json" -Raw).IndexOf("{Your Sub Id}")
```

Comment:
Could we add a quick summary to the top of this.  Maybe make a note that it is importing the role at the subscription level for use later?